src/prompt/choices.ts

```diff
@@ -0,0 +1,13 @@
+import { Tile, flatternTiles } from "../tile/tile";
+
+export const generateChoices = (gameConfig) =>
+  flatternTiles(gameConfig).reduce((acc, tile: Tile) => {
+    if (tile != undefined && tile.symbol !== "O") {
```

Comment:
```suggestion
    if (tile != undefined && tile.symbol !== NEUTRALE_TILE.symbol) {
```

To make it more meaningful :wink: (hello @arimet :wave: )

Reply:
Oops... magic string... Done